app/controllers/application_controller.rb

```diff
@@ -10,7 +10,20 @@ class ApplicationController < ActionController::Base
     strategy DecentExposure::StrongParametersStrategy
   end
 
+  private
+
   def require_gds_editor_permissions!
     authorise_user!("GDS Editor")
   end
+
+  # FIXME: clean this up when we're using content_ids in the URL.
+  def find_topic_for_sector_id
```

Comment:
I agree that would probably be the best place for it.  However, I plan to shortly change this to not use these slugs in the URLs, so this will go away very soon.

Happy to update this if you think it's worth it...